AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/InitialSync/InitialSyncOrchestrator.swift

```diff
@@ -102,9 +114,27 @@ final class AWSInitialSyncOrchestrator: InitialSyncOrchestrator {
                                                        dataStoreConfiguration: dataStoreConfiguration,
                                                        completion: syncOperationCompletion)
 
+        initialSyncOperationSinks[modelType.modelName] = initialSyncForModel
+            .publisher
+            .sink(receiveCompletion: onReceiveCompletion(completed:),
+                  receiveValue: onReceiveValue(receiveValue:))
+
         syncOperationQueue.addOperation(initialSyncForModel)
     }
 
+    private func onReceiveCompletion(completed: Subscribers.Completion<DataStoreError>) {
+        switch completed {
+        case .finished:
+            initialSyncOrchestratorTopic.send(completion: .finished)
+        case .failure(let error):
+            initialSyncOrchestratorTopic.send(completion: .failure(error))
+        }
+    }
```

Comment:
Oh, actually, `func onReceiveCompletion()` will never be called since `completion` is never sent from `InitialSyncOperation`. 

But in this case, maybe I should take failure case into consideration maybe some error may occurs in `InitialSyncOperation`?